src/test/scala/com/cognite/sdk/scala/v1/SequencesTest.scala

```diff
@@ -148,10 +161,16 @@ class SequencesTest extends SdkTestSpec with ReadBehaviours with WritableBehavio
   it should behave like updatableById(
     client.sequences,
     sequencesToCreate,
-    Seq(SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-1-1"))), SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-2-1")))),
+    Seq(
+      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-1-1"))),
+      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-2-1")), dataSetId = Some(SetValue(testDataSet.id))),
+      SequenceUpdate(name = Some(SetValue("scala-sdk-write-example-3-1")), dataSetId = Some(SetNull()))
+    ),
     (readSequences: Seq[Sequence], updatedSequences: Seq[Sequence]) => {
       assert(readSequences.size == updatedSequences.size)
       assert(updatedSequences.zip(readSequences).forall { case (updated, read) =>  updated.name.get == s"${read.name.get}-1" })
+      val dataSets = updatedSequences.map(_.dataSetId)
+      assert(List(None, Some(testDataSet.id), None) === dataSets)
```

Comment:
`dataSets should contain inOrderOnly (None, Some(testDataSet.id), None)`
http://www.scalatest.org/user_guide/using_matchers#workingWithSequences

Reply:
That does not work, it says `inOrderOnly must not contain any duplicated values` :/ Why do you think it'd be better than equality?